Documentation/ApiOverview/Fal/UsingFal/Frontend.rst

```diff
@@ -71,7 +71,8 @@ Example:
 ..  note::
     The system extension `filemetadata`_ (if installed) provides some additional
     meta data fields for files, for example :html:`creator`, :html:`publisher`,
-    :html:`copyright` and others. To access those fields in the frontend, you
+    :html:`copyright` and others. To access those fields
+    :ref:`(see list) <fal-architecture-database-sys-file-metadata>` in the frontend, you
     have to use a proxy method named :html:`properties`:
```

Comment:
There is not method `properties`. There is a method called `getProperties()` which can be called via `properties` in Fluid Template

Reply:
This is a very important note, yes! I'll also improve this (was already there originally, not in my patch)